package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "connector-registry-storage-adapter",
+  "version": "1.0.0",
+  "description": "Adapter for Connector Registry github storage",
+  "main": "dist/index.js",
+  "scripts": {
+    "build": "tsc --build",
+    "clean": "tsc --build --clean",
+    "test": "echo \"Error: no test specified\" && exit 1"
```

Comment:
Can you add tests please?
For example, for `jest` you can use the following settings
```
  "jest": {
    "rootDir": ".",
    "coverageThreshold": {
      "global": {
        "branches": 100,
        "lines": 100,
        "statements": 100
      }
    },
    "collectCoverage": true,
    "testPathIgnorePatterns": [
      "<rootDir>/test/__fixtures__/"
    ],
    "coveragePathIgnorePatterns": [
      "<rootDir>/test/__fixtures__/"
    ],
    "reporters": [
      "default",
      "jest-junit"
    ],
    "testEnvironment": "node",
    "setupFilesAfterEnv": [
      "./jest.setup.js"
    ],
    "resolver": "<rootDir>/test/resolver.js"
  }
```
See https://github.com/adobe/aio-cli-plugin-app-templates/blob/6bb78b936d8a0d5c4ea32a9a96664810fdc6c129/package.json#L78-L103